tools/challenge-helper-scripts/create-next-task.ts

```diff
@@ -0,0 +1,52 @@
+import ObjectID from 'bson-objectid';
+import { getTemplate } from './helpers/get-challenge-template';
+import { newTaskPrompts } from './helpers/new-task-prompts';
+import { getProjectPath } from './helpers/get-project-info';
+import {
+  getMetaData,
+  updateMetaData,
+  validateMetaData
+} from './helpers/project-metadata';
+import {
+  createChallengeFile,
+  updateTaskMeta,
+  updateTaskMarkdownFiles
+} from './utils';
+
+const createNextTask = async () => {
+  validateMetaData();
+
+  const { challengeType } = await newTaskPrompts();
+
+  const options = {
+    title: `Task 0`,
+    dashedName: 'task-0',
+    challengeType
+  };
```

Comment:
```suggestion
  // Placeholder title, to be replaced by updateTaskMarkdownFiles
  const options = {
    title: `Task 0`,
    dashedName: 'task-0',
    challengeType
  };
```
Just because I wasn't sure why the title needed to exist.